crankshaft-engine/src/service/runner.rs

```diff
@@ -107,7 +108,8 @@ impl Runner {
 
         tokio::spawn(async move {
             let _permit = lock.acquire().await?;
-            let result = backend.clone().run(task, Arc::new(|_| {}), token)?.await;
+            let (started_tx, _) = oneshot::channel();
```

Comment:
Why are we ignoring the rx here?

Reply:
The runner doesn't do anything with the notification of tasks being started; alternatively we could make this `Option<oneshot::Sender<()>>` and pass in `None` here.

Reply:
Oh okay, I see now after looking at the code. My next question would have been "well then why do we have it at all if the only caller of `Backend::run()` is `Runner::spawn()`", but I see now that it's possible _someone_ would want it (an API consumer of `crankshaft` working with the `Backend` trait directly?). I'm okay with keeping it.

Reply:
I'll change to `Option` since the backends wrap it in `Option` anyway internally.

Reply:
Yes, we talk to the backend directly in `wdl-engine` and don't use a `Runner`.